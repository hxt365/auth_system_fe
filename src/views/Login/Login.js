import React from 'react';
import LoginForm from 'components/LoginForm';
import Logo from 'assets/pictures/logo.png';
import './Login.scss';

function Login() {
  return (
    <section className="login">
      <img className="login__logo" src={Logo} alt="Teko's logo" />
      <h1>Come in with us!</h1>
      <LoginForm />
    </section>
  );
}

export default Login;
