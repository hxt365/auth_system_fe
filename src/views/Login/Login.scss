.login {
  text-align: center;

  &__logo {
    width: 35rem;
    margin-bottom: -4rem;
  }
}
