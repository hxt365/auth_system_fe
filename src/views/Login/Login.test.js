import React from 'react';
import toJson from 'enzyme-to-json';
import { shallow } from 'enzyme';
import Login from './Login';

describe('test Login', () => {
  it('test snapshot', () => {
    const wrapper = shallow(<Login />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
