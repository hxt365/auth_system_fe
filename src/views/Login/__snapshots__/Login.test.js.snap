// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Login test snapshot 1`] = `
<section
  className="login"
>
  <img
    alt="Teko's logo"
    className="login__logo"
    src="logo.png"
  />
  <h1>
    Come in with us!
  </h1>
  <Form(LoginForm) />
</section>
`;
